ya_gpt_bot/services/impl/conversation_service.py

```diff
@@ -0,0 +1,101 @@
+"""Service to get and update conversations."""
+import datetime
+from typing import Callable
+
+from sqlalchemy import delete, func, insert, select
+from sqlalchemy import text as sa_text
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.ext.asyncio import AsyncEngine
+
+from ya_gpt_bot.db.entities.conversation import t_conversation
+
+func: Callable
+
+MIN_TS_SQL = sa_text(
+    """
+with agg as (
+    select
+        user_from || ',' || coalesce(user_to, '') || ',' || text as full_message,
+        message_timestamp as ts
+    from
+        ya_gpt_bot.public.conversation c
```

Comment:
`ya_gpt_bot` part is definitely should not be set here as database name may differ as it is defined in config

Reply:
migrated string sqls to orm